packages/twenty-front/src/generated/graphql.tsx

```diff
@@ -1291,6 +1291,12 @@ export type TransientToken = {
   transientToken: AuthToken;
 };
 
+export type TrialPeriod = {
```

Comment:
we probably have to also generate the metadata (same command but configuration metadata instead of data)